context/CocktailContext.tsx

```
@@ -9,18 +9,14 @@ import {
   useMemo,
 } from "react";
 import type { ReactNode } from "react";
-import {
-  requestCocktailRecommendation,
-  type Cocktail,
-  type BartenderRequest,
-} from "@/api/cocktail";
+import type { Cocktail, BartenderRequest } from "@/api/cocktail";
 import {
   asyncStorage,
   saveToStorageAsync,
```

ESLint

[error] 'saveToStorageAsync' is defined but never used.

(@typescript-eslint/no-unused-vars)